src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -71,36 +159,78 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 return;
             }
 
+            // NOTE: Values in TagHelperOutput.Attributes are already HtmlEncoded
+
             var content = new StringBuilder();
+            
+            if (modeResult.Mode == Mode.Fallback && string.IsNullOrEmpty(HrefInclude))
```

Comment:
What about: 
```
<link rel="stylesheet" 
          asp-fallback-href-include="**/blank.css,**/site.css"
          asp-fallback-test-class="fallback-test"
          asp-fallback-test-property="visibility"
          asp-fallback-test-value="hidden" />
```

Doesn't this require us to glob link tags? I might be missing something.